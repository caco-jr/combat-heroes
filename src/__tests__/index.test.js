import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';

import App from '../pages/index';
import makeStore from '../redux';

describe('With Snapshot Testing', () => {
  it('App shows "Hello world!"', () => {
    const component = renderer.create(
      <Provider store={makeStore()}>
        <App />
      </Provider>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
