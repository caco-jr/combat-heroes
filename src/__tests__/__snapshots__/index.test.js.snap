// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
Array [
  <header
    className="container"
  >
    Combat Heroes
  </header>,
  <section
    className="indexstyle__PreviewBackground-j30b2b-0 eVujOF"
  >
    <section
      className="indexstyle__PreviewWrapper-j30b2b-1 jmliuo container"
    >
      <section
        className="indexstyle__PreviewSlotWrapper-sc-1juzzlq-0 kiJQsP"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          Aleatório
        </button>
      </section>
      <section
        className="indexstyle__PreviewSlotWrapper-sc-1juzzlq-0 kiJQsP"
      >
        <button
          onClick={[Function]}
          type="button"
        >
          Aleatório
        </button>
      </section>
    </section>
  </section>,
  <section
    className="container"
  >
    <form
      className="indexstyle__SearchWrapper-sc-1r0ek5c-0 hEclBY"
      onSubmit={[Function]}
    >
      <input
        className="indexstyle__SearchInput-sc-1r0ek5c-1 eJzVLt"
        name="search"
        onChange={[Function]}
        value=""
      />
    </form>
  </section>,
  <section
    className="container"
  >
    <h2>
      Most Popular
    </h2>
    <section
      className="indexstyle__CardList-sc-5v81d9-0 eIwYVw"
    />
  </section>,
]
`;
